manifests/params.pp

```diff
@@ -13,6 +13,7 @@
   # by cvmfs. If you are managing autofs elsewhere set to false.
   # This being the string 'true' and not boolean true is a hiera bug.
   $config_automaster      = hiera('cvmfs_config_automaster','true')
+  $manage_autofs_service  = hiera('cvmfs_manage_autofs_service','true')
```

Comment:
Please set this just *true* and skip the explicit hiera call. I plan to deprecate all these hiera calls in favor of parameter lookup.